components/src/dynamo/thompson_router/config.py

```
@@ -0,0 +1,50 @@
+
+def load_config() -> dict[str, Any]:
+    """Load Thompson router config, preferring env override over bundled default."""
+    override_path = os.environ.get(_ENV_OVERRIDE)
+
+    if override_path:
+        try:
+            with open(override_path, encoding="utf-8") as f:
+                config = yaml.safe_load(f) or {}
+            logger.info("Loaded Thompson config from %s", override_path)
+            return config
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)